pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -53,6 +54,9 @@ const (
 	DefaultCNIDir     = "/opt/cni/bin"
 
 	sysctlBridgeCallIptables = "net/bridge/bridge-nf-call-iptables"
+
+	// private mac prefix safe to use
```

Comment:
my point was that a comment is needed :)

On Fri, Aug 19, 2016 at 4:50 PM, Girish Kalele <notifications@github.com>
wrote:

> In pkg/kubelet/network/kubenet/kubenet_linux.go
> <https://github.com/kubernetes/kubernetes/pull/28717#discussion_r75566276>
> :
>
> > @@ -53,6 +54,9 @@ const (
> >  	DefaultCNIDir     = "/opt/cni/bin"
> >
> >  	sysctlBridgeCallIptables = "net/bridge/bridge-nf-call-iptables"
> > +
> > +	// private mac prefix safe to use
>
> Ethernet OUI assignments http://standards-oui.ieee.org/oui.txt
>
> Ethernet OUIs are supposed to be 6 nibbles though.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/28717/files/c8992c0a2bcfb216c073c03ff6cc551915f99003#r75566276>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFVgVMk6o95cEhsl1X3DEEnpu2Ukbn-Zks5qhkFOgaJpZM4JIcoW>
> .
>